src/components/LazyLog/index.jsx

```diff
@@ -448,24 +545,68 @@ export default class LazyLog extends Component {
     return <Loading />;
   };
 
+  renderSearchBar = () => {
+    const { enableSearch } = this.props;
+
+    if (enableSearch) {
```

Comment:
It doesn't look like the `enableSearch` prop is used anywhere else but here, so feel free to remove the destructuring and inline it:

```js
if (this.props.enableSearch) {
```